src/qiskit_sphinx_theme/theme/qiskit-sphinx-theme/partials/icons.html

```diff
@@ -10,41 +10,33 @@
     * svg-arrow-right
     * svg-pencil
     * svg-eye
-
-  The SVGs downloaded from the Carbon website were edited to remove a transparent rectangle and simplify formatting.
```

Comment:
Could you please add a note about how the icons were generated? So that future people can reproduce it.

Also is the list above still accurate of what we replace and what we don't?

Reply:
I added the note. Yes, the list is still accurate.